tests/foreman/sys/test_fam.py

```diff
@@ -33,10 +50,6 @@ def test_positive_ansible_modules_installation(default_sat):
         available and supported modules are contained
 
     """
-    result = default_sat.execute(
-        'yum install -y ansible-collection-redhat-satellite --disableplugin=foreman-protector'
-    )
-    assert result.status == 0
```

Comment:
I'm not sure if things have changed between Satellite versions or our install process, but any of the VMs I've tested with already have FAM installed.

Reply:
If we no longer need to run yum, I suppose you can remove @pytest.mark.destructive